config/settings.py

```diff
@@ -44,6 +44,10 @@
 # For removing inactive users from groups
 NUMBER_OF_INACTIVE_MONTHS_UNTIL_REMOVAL_FROM_GROUP_NOTIFICATION = 6
 NUMBER_OF_DAYS_AFTER_REMOVAL_NOTIFICATION_WE_ACTUALLY_REMOVE_THEM = 7
+# Set the default theme for new groups: GENERAL, FOODSAVING, BIKEKITCHEN
+GROUP_THEME_DEFAULT = grouptypes.GroupTheme.FOODSAVING
+# Set the default status for new groups, ACTIVE, PLAYGROUND, INACTIVE
+GROUP_STATUS_DEFAULT = grouptypes.GroupStatus.ACTIVE
```

Comment:
Why did you move that as well?

Reply:
I didn't move it, I want to guarantee that the variables are set and have a default value that can be overwritten by local_settings.py and that they are not hardcoded in the manager or in models.py (however, it might be better to have them in models.py …)

Reply:
The problem with that then is that they *have* to be present in local_settings.py … so I wanted to guarantee that they are present by adding them to settings.py

Reply:
I mean why you made the default GroupStatus into a setting?